docs/requirement-sets/common/device-permission-service-requirement-sets.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Device Permission Service requirement sets
+description: Learn more about the Device Permission Service API requirement sets and the platforms it supports.
+ms.date: 05/13/2024
+ms.topic: overview
+ms.localizationpriority: medium
+---
+
+# Device Permission Service requirement sets
+
+Requirement sets are named groups of API members. Office Add-ins use requirement sets specified in the manifest or use a runtime check to determine whether an Office application supports APIs that an add-in needs. For more information, see [Office versions and requirement sets](/office/dev/add-ins/develop/office-versions-and-requirement-sets).
+
+With the Device Permissions Service requirement set, your add-in running in Office on the web can request access to a user's device capabilities. A user's device capabilities include their camera, geolocation, or microphone.
```

Comment:
Omit the web limitation in case the API is expanded later. The table makes the platform restriction very clear.
```suggestion
With the Device Permissions Service requirement set, your add-in can request access to a user's device capabilities. A user's device capabilities include their camera, geolocation, or microphone.
```